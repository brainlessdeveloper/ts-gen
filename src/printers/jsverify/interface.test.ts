import ts from 'typescript'

import printJscRecord from './interface'

const resultFile = ts.createSourceFile(
  '', // File name.
  '', // Source text.
  ts.ScriptTarget.Latest, // Language version.
  false, // Set parent nodes.
  ts.ScriptKind.TS, // JS, TS, etc.
)

describe('jsverify record printer', () => {
  const result = printJscRecord(resultFile)

  it('produces a result', () => {
    expect(result).toMatchSnapshot()
  })
})
