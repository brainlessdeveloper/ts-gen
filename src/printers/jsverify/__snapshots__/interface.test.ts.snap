// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jsverify record printer produces a result 1`] = `
"jsc.record({
    field: true
});"
`;
